### ISSUE TITLE
Updating Phoscon SW breaks communication with OpenHab

### ISSUE THREAD

**Issue by FrankH1** [openhab/openhab-addons#10390]
Running this environment for more than a year now, runs quite stable. Several aktors connected via Wifi/MQTT and several sensors/aktors connected via ZigBee and the Conbee Gateway.
Did face that problem already last year when upgrading the Conbee-SW to 2.05.75 - the connection with OH broke and I did see permanent online/offline changes of the Gateway - (see also https://github.com/dresden-elektronik/deconz-rest-plugin/issues/2563)
The solution was at that time to go back to Conbee-SW 2.05.74
Meanwhile there are a few sensors I want to connect via Zigbee - this should be supported by newer SW releases of the Conbee stick and is not supported by the version I'm currently using. Unfortunately I get the same issues after updating the SW - connection breaks and the log-file flows over with online/offline changes. Did report this on the deconz-rest plugin tree (https://github.com/dresden-elektronik/deconz-rest-plugin/issues/4638) but the advice was that this must be an OH- problem.
Running out of logical explanations - clearly also driven by the fact that I'm not familiar with the plugin - concept. Clearly that is a problem which needs to be solved between the plugin and the deconz - REST API- but clearly as well there is probably a huge number of users where this environment works.
Any hint??

## Steps to Reproduce (for Bugs)

1.
2.

## Your Environment

* Version used: openHAB 2.5.0 on RPI3; * deconz binding 2.5.0; hue binding 2.5.0
* Conbee II on the same RPI with FW 2680700 and 2.05.74
* Environment name and version (e.g. Chrome 76, Java 8, Node.js 12.9, ...):
* Operating System and version (desktop or mobile, Windows 10, Raspbian Buster, ...):

**Comment by J-N-K** [comment]
Try latest 2.5.x version. I'm pretty sure I fixed that in 3.x, not sure if it was backported to 2.5.x

**Comment by FrankH1** [comment]
> Try latest 2.5.x version. I'm pretty sure I fixed that in 3.x, not sure if it was backported to 2.5.x

Hi, thanks a lot. I was even not aware that after 2.5.1 came something :-(
Did update to 2.5.12 and ended in a complete nightmare with the Conbee.
OH seems to work properly - I was only faced with one new errormsg ( [WARN ] [ng.exec.internal.handler.ExecHandler] - Tried to execute 'python /home/openhabian/meinbme280.py', but it is not contained in whitelist) but I managed this meanwhile with adding it to the whitelist).
Conbee II made me mad. I did update to the newest Firmware/Software. Was happy that the behaviour of going online/offline was gone. But the Conbee network lost everything and sensors were offline and I couldnt bring them back. Tried several combinations of FW for the Stick and deconz with Resets, reloading config etc - I was ready to throw everything away. 
Finally it works; I had to add a few sensors and lights again - no idea what was going on. Its like our easter-break story in Germany - quite confusing and mystic.
But - The interface between OH and deconz is working stable even with newer FW - that was the issue and it is solved! Thanks for the hint. 
So issue seems to be closed - I dont expect that this jumps up again over night.
